pkg/stockings/stock.go

```diff
@@ -18,6 +19,10 @@ type StockAPI interface {
 	GetHistoricalRange(ticker string, start time.Time, end time.Time) (*HistoricalStocks, error)
 }
 
+const (
+	DateLayout = "20060102"
```

Comment:
this date format is only applicable to iex api, so can you put it under `iex.go`?

Reply:
this is used in a global scope so I thought it might as well be in stock.go

Reply:
done, just noting that fingo also uses this date format